modules/module2/slides/module2_03.qmd

```diff
@@ -191,7 +174,7 @@ The slides show an example of what this syntax looks like in general.
 Since we have uploaded our dataset to the course online repository,
 we will read the data directly from there
 using this URL
-<https://raw.githubusercontent.com/UBC-MDS/exploratory-data-viz/main/chapters/en/slides/module2/data/world-data-gapminder.csv>.
+<https://raw.githubusercontent.com/UBC-MDS/exploratory-data-viz/refs/heads/quarto_website/modules/data/world-data-gapminder.csv>.
```

Comment:
This need to refer to the main branch, otherwise it will not work when it is merged in. It is ok if it doesn't work now because of that, so change it to the path where it will be when merged into main.